### PR TITLE
Properly add prickle and smolder labels (fixes #2636)

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

**Review by kotp — APPROVED** [review]

File: .appends/.github/labels.yml [hunk 1]
```diff
@@ -174,3 +174,11 @@
...
   description: "I have a proposal or question about things, but nothing is broken."
   color: "293028"
+
+- name: "prickle💢"
+  description: "This issue should be skipped by the auto-responder."
+  color: "FFFFFF"
+
+- name: "smolder🍲"
+  description: "Lower-priority and/or future idea"
+  color: "8B0000"
```
Comment (kotp) [inline .appends/.github/labels.yml:184]: Should these be in alphabetical order?  The others seem to be organized this way.

**Comment by BethanyG** [comment]
Can we hold this until I take a look at the rest of the appended labels?  _Many thanks._

**Comment by SaschaMann** [comment]
I would suggest merging this earlier and then removing them later on if desired. Otherwise the sync PR will remove them and updating that will be more annoying.

**Comment by ErikSchierboom** [comment]
As Sascha said, I'd merge this PR as it is merely fixing something that is currently wrong. I'd look at the appended labels later.

**Review by BethanyG — APPROVED** [review]
Approving, and will look at the file later.

**Comment by ErikSchierboom** [comment]
Awesome! Thanks for the speedy merge

**Pull request merged**